onnxruntime/core/graph/schema_registry.cc

```diff
@@ -194,6 +194,23 @@ DomainToVersionMap SchemaRegistryManager::GetLatestOpsetVersions(bool is_onnx_on
   return domain_version_map;
 }
 
+static bool IsDomainVersionBeyondSupportedRange(
+    const std::string& domain,
+    const int op_set_version) {
+  // check the ONNX schema registry
+  auto& onnx_domain_version_map =
```

Comment:
> auto& [](start = 1, length = 6)

Prefer to specify constness explicitly even with auto

Reply:
this is from change ([https://github.com/microsoft/onnxruntime/pull/2413](https://github.com/microsoft/onnxruntime/pull/2413)) and we are just pulling in the latest version to our branch. The change is already in master. If it needs to be addressed, please open a bug to the person who changed this.

---
In reply to: [350341673](https://github.com/microsoft/onnxruntime/pull/2470#discussion_r350341673) [](ancestors = 350341673)